src/InteractiveMars/Mars.js

```diff
@@ -0,0 +1,62 @@
+/* TO-DO 
+- Make marker onClick zoom less janky (or disable alltogether)
+- Label markers with rover names without having to hover? (ThreeJS text geometry?)
+- Make enter button look pretty
+*/
+
+import React, { useState } from "react";
+import ReactGlobe from "react-globe";
+import { Markers } from "./Markers";
+import "tippy.js/dist/tippy.css";
+import "tippy.js/animations/scale.css";
+
+//Function to render onHover tooltips
+function markerTooltipRenderer(marker) {
+  return `ROVER: ${marker.rover} `;
+}
+
+//Options for globe, more options found at https://react-globe.netlify.app/usage/options
+const options = {
+  markerTooltipRenderer,
+  cameraAutoRotateSpeed: 0.8,
+  cameraMaxPolarAngle: (Math.PI * 9) / 16,
+  cameraMinPolarAngle: (Math.PI * 7) / 16,
+  cameraRotateSpeed: 0.5,
+  enableCameraAutoRotate: true,
+  enableCameraRotate: true,
+  enableCameraZoom: false
+};
+
+//Globe export
+
+export function Mars() {
+  // Get marker data from Markers.js
+  const [markers] = useState(Markers);
+
+  return (
+    <div>
+      {/* Enter mars button */}
+      <form style={{
+        position: 'absolute', left: '50%', top: '50%',
+        transform: 'translate(-50%, -50%)'
+      }}
+        action="http://localhost:3000/rovers" method="get">
+        <button>ENTER MARS</button>
+      </form>
```

Comment:
This whole thing is just to take the user to a new page.
As a result, it shouldn't be a form and a button - links do this much more easily!